orbit/utils/dataset.py

```diff
@@ -0,0 +1,52 @@
+import pandas as pd
+
+def load_iclaims():
+    """
+    Load iclaims dataset
+
+    Returns
+    -------
+        pandas DataFrame
+
+    Notes
+    -----
+    https://fred.stlouisfed.org/series/ICNSA
+    """
+    url = 'https://raw.githubusercontent.com/uber/orbit/master/examples/data/iclaims_example.csv'
+    df = pd.read_csv(url)
```

Comment:
Somehow I always need to solve the SSL problem otherwise i get this error:
```
URLError: <urlopen error [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:852)>
```

Reply:
to resolve this, one could use 
```
import ssl
ssl._create_default_https_context = ssl._create_unverified_context
```
however, this will add the dependency of `ssl` to our package. This is something we dont like. maybe we shouldn't babysit the users all the time.